python/lsst/drp/tasks/assemble_cell_coadd.py

```diff
@@ -76,6 +82,15 @@ class AssembleCellCoaddConnections(
         multiple=True,
     )
 
+    visitSummaryList = Input(
+        doc="Input visit-summary catalogs with updated calibration objects. Mainly used for coadd weights.",
+        name="finalVisitSummary",
+        storageClass="ExposureCatalog",
+        dimensions=("instrument", "visit"),
+        deferLoad=True,
+        multiple=True,
+    )
```

Comment:
One potential downside here is that it opens up `AssembleCellCoadd` to the Piff memory leak.  I'm not sure if that's a problem at all, but since it'd be all PSFs for all detectors in any visit being coadded, it might be a larger number of those PSFs in memory than we typically have in a single quantum.

An alternative might be the parquet `ccdVisitTable`.  Thus far we haven't used that as an input to anything other than analysis tasks, but I don't think there's any reason we couldn't.

Reply:
I ran a test run over several hundred patches without any trouble, so I doubt that this is an issue. But `ccdVisitTable` was my first thought but changed it to `finalVisitSummary` after we discussed about it. I'd be happy to switch it to `ccdVisitTable` and may be even read just the column corresponding to `meanVar`.

Reply:
Relying on a parquet table for weights/mean variance has the advantage that I can mock this up easliy in unit tests, and it's possible for simulations running this code outside of normal pipeline/middleware workflow to provide in a replacement.